### ISSUE TITLE
None of the Package Examples work with go mod tidy

### ISSUE THREAD

**Issue by drgrib** [browserutils/kooky#73]
Running `go mod tidy` for the first example, I get

```
go: finding module for package github.com/browserutils/kooky/browser/all
go: finding module for package github.com/browserutils/kooky
go: found github.com/browserutils/kooky in github.com/browserutils/kooky v0.2.0
go: test.test imports
        github.com/browserutils/kooky: github.com/browserutils/kooky@v0.2.0: parsing go.mod:
        module declares its path as: github.com/zellyn/kooky
                but was required as: github.com/browserutils/kooky
```

For the second one, I get
```
go: finding module for package github.com/browserutils/kooky/browser/chrome
go: test.test imports
        github.com/browserutils/kooky/browser/chrome: module github.com/browserutils/kooky@latest found (v0.2.0), but does not contain package github.com/browserutils/kooky/browser/chrome
```

For the third one, I get
```
go: finding module for package github.com/browserutils/kooky/browser/safari
go: test.test imports
        github.com/browserutils/kooky/browser/safari: module github.com/browserutils/kooky@latest found (v0.2.0), but does not contain package github.com/browserutils/kooky/browser/safari
```

I suspect the recent changes to your go.mod file in #71 are incomplete have made this package unusable for new users.

**Comment by drgrib** [comment]
Further confirmation that this package is in a bad state. Perhaps it just needs a new release:
```
>> go get github.com/browserutils/kooky
go: github.com/browserutils/kooky@v0.2.0: parsing go.mod:
        module declares its path as: github.com/zellyn/kooky
                but was required as: github.com/browserutils/kooky
```
```
>> go get github.com/browserutils/kooky@d4f81abd0200477c02a2745278fdb7bb3ce0387d
go: github.com/browserutils/kooky@v0.2.0: parsing go.mod:
        module declares its path as: github.com/zellyn/kooky
                but was required as: github.com/browserutils/kooky
```

**Comment by alxjsn** [comment]
@drgrib the problem is that the fix in #71 was never included in a new tagged release.

This should work: `go get github.com/browserutils/kooky@master`, but it would be nice if @zellyn could tag a new release.

**Comment by srlehn** [comment]
published v0.2.1 - it should work again

**Comment by srlehn** [comment]
all package examples (Example_chromeSimpleMacOS, Example_cookieJar), regular examples (ExampleReadCookies_all, ExampleFindAllCookieStores, ExampleExportCookies, ExampleFilter_regex, ExampleFilterCookies) functions and readme examples (Any Browser - Cookie Filter Usage, Chrome on macOS, Safari) tested.
